build-final.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+IMAGE=threema/webrtc-build-tools:latest
+TARGETS="${WEBRTC_TARGETS:-arm arm64 x86 x64}"
+BUILD_ARGS="${WEBRTC_BUILD_ARGS:-symbol_level=1 enable_libaom=false}"
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <revision>"
+    echo "Example: $0 branch-heads/4430"
+    exit 1
+fi
+revision=$1
+
+mkdir -p out && chmod 777 out
+docker run --rm -ti -v "$(pwd)/out:/out" -v "$(pwd)/patches:/patches" \
+    $IMAGE /bin/bash -c "
+    set -euo pipefail
+    shopt -s nullglob
+
+    export GYP_DEFINES='OS=android'
```

Comment:
I have removed this from the local builds and it seems to make no difference. We're now using GN instead of GYP, so I don't think it does anything.

Reply:
Hm, that's a good point.

Reply:
Removed.